packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -1157,27 +1157,51 @@ class CupertinoDynamicColor with Diagnosticable implements Color {
     }
   }
 
+  @Deprecated(
+    'Use component accessors like .r or .g, or toARGB32 for an explicit conversion. '
+    'This feature was deprecated after v3.33.0-1.0.pre.',
+  )
   @override
   int get value => _effectiveColor.value;
 
   @override
-  int toARGB32() => value;
+  int toARGB32() => _effectiveColor.toARGB32();
 
+  @Deprecated(
+    'Use (*.a * 255.0).round().clamp(0, 255). '
+    'This feature was deprecated after v3.33.0-1.0.pre.',
+  )
   @override
   int get alpha => _effectiveColor.alpha;
 
+  @Deprecated(
+    'Use (*.b * 255.0).round().clamp(0, 255). '
+    'This feature was deprecated after v3.33.0-1.0.pre.',
+  )
   @override
   int get blue => _effectiveColor.blue;
 
   @override
   double computeLuminance() => _effectiveColor.computeLuminance();
 
+  @Deprecated(
```

Comment:
++ for a dart fix for this one

Reply:
Oh wait I just remembered why we don't have fixes for those since they are too complex to be helpful. I will correct my comments!